pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,213 @@
+    </td>
+  </tr>
+  <tr>
+    <td><code>steps</code></td>
+    <td>
+    A list of steps in the group; at least 1 step is required. Allowed step types: `wait` and `trigger`.<br>
+      <em>Type:</em> <code>array</code>
+    </td>
+  </tr>
+</table>
+
+_Optional attributes:_  
+
+<table data-attributes>
+  <tr>
+    <td><code>allow_dependency_failure</code></td>
+    <td>
+      Whether to continue to run this step if any of the steps named in the <code>depends_on</code> attribute fail.<br>
+      <em>Default:</em> <code>false</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this step in their builds.<br>
+      <em>Example:</em> <code>"main stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>depends_on</code></td>
+    <td>
+      A list of step keys that this step depends on. This group will only run after the named steps have completed. See <a href="/docs/pipelines/dependencies">managing step dependencies</a> for more information.<br>
+      <em>Example:</em> <code>"test-suite"</code>
+    </td>
+   </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A map of <a href="/docs/pipelines/environment-variables">environment variables</a> for this step.<br>
+      <em>Example:</em> <code>RAILS_ENV: "test"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>if</code></td>
+    <td>
+      A conditional to whether or not this group should be included in the build. A boolean expression that omits the step when false. See <a href="/docs/pipelines/conditionals">Using conditionals</a> for supported expressions.<br>
+      <em>Example:</em> <code>build.message != "skip me"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>key</code></td>
+    <td>
+    	A unique string to identify the block step. Alias: `identifier` or `id`.
+      <em>Example:</em> <code>"test-suite"</code>
+    </td>
+  </tr>  
+  <tr>
+    <td><code>label</code></td>
+    <td>
+      The label that will be displayed in the pipeline visualisation in Buildkite (name of the group in the UI). Supports emoji.<br>
+      <em>Example:</em> <code>"\:hammer\: Tests" will be rendered as "🔨: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>skip</code></td>
+    <td>
+      Whether to skip this group or not. Passing a string provides a reason for skipping this command. Passing an empty string is equivalent to <code>false</code>.
+      Note: Skipped steps will be hidden in the pipeline view by default, but can be made visible by toggling the 'Skipped jobs' icon.<br>
+      <em>Example:</em> <code>true</code><br>
+      <em>Example:</em> <code>false</code><br>
+      <em>Example:</em> <code>"My reason"</code>
+    </td>
+  </tr>
+</table>
+
+## Parallel groups  
+
+If you have two group steps next to each other, they run in parallel. For example:  
```

Comment:
What is YAMLese for next to each other? Although, as you can't nest steps, this is just if you have two groups, right?

Reply:
Do you want to test what happens if you have a command in between the two groups?

Reply:
I think the "YAMLese" is "two (or more? I need to find out) group steps consecutively defined in a YAML config", but I do need to find out if this is still going to be valid with a command thrown in between them.